ios/AudioInputController.m

```diff
@@ -68,8 +83,11 @@ - (OSStatus) prepareWithSampleRate:(double)desiredSampleRate {
 
         NSLog(@"[WebRTCVad] set mode %d", ok);
 
-        // 5 second audio buffer hint
-        [audioSession setPreferredIOBufferDuration:5 error:nil];
```

Comment:
Documentation for `setPreferredIOBufferDuration`

> The typical maximum I/O buffer duration is 0.093 seconds (corresponding to 4,096 sample frames at a sample rate of 44.1 kHz). The minimum I/O buffer duration is at least 0.005 seconds (256 frames) but might be lower depending on the hardware in use.

This function only sets your preference, but the OS itself still decides ultimately what to send you. I bet that 5 seconds was always invalid, butthe OS was handling it more gracefully or with a more compatible buffer size before.